packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -9,7 +9,7 @@ import {
   ChannelService,
   getRootWorkspacePath,
   SFDX_CORE_CONFIGURATION_NAME,
-  TelemetryService
+  TelemetryService,
```

Comment:
hum...I was previously the eslint/prettier upgrade branch and I think a new formatter stuck around when I switched branched...I'll fix the comma issues.